goss-testing/tests/common/goss-network-interfaces.yaml

```diff
@@ -22,8 +22,33 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 # Variable set: network_interfaces
+#
```

Comment:
It looks like you added in another license header

Reply:
Fixed ☹️